src/centreon/common/airespace/snmp/mode/radiusauthservers.pm

```diff
@@ -54,9 +54,9 @@ sub set_counters {
     $self->{maps_counters}->{servers} = [
         { label => 'authserver-roundtrip-time', nlabel => 'radius.authserver.roundtrip.time.milliseconds', set => {
                 key_values => [ { name => 'round_trip_time' }, { name => 'name' } ],
-                output_template => 'round trip time: %s ms',
+                output_template => 'round trip time: %s µs',
```

Comment:
Should we make a breaking change for this @omercier  ?

Reply:
It will "break" the curves but not the configuration, so we don't consider it as a breaking change